Brocfile.js

```diff
@@ -15,6 +16,12 @@ function treeGenerator (dir) {
     }
 };
 
+var bindTree = new Funnel('node_modules/phantomjs-polyfill', {
```

Comment:
I'm going to keep this around for a bit, not needed in CI but most people aren't on phantomjs 2